### ISSUE TITLE
'style' option has no effect

### ISSUE THREAD

**Issue by mooisjken** [artem-sedykh/mini-climate-card#37]
For the 'buttons' option, setting a style (e.g. a color) only changes the 'off' color, not the 'on' color ('on' meaning when the linked entity is enabled). It seems that the 'on' color takes precedence over the style color. This could be fixed in 2 ways:

1. setting the priority of the 'style' color above the 'active' color. It will then be the user's responsibility to handle the color logic (including the 'disabled' color)
2. by having an 'active' field for buttons as well (similar to hvac_mode or dropdowns), which would enable the user to set "active: false" so that the 'off' color will show each time

For 'hvac_mode', using the style option has no effect at all, even when setting "active: false".

**Comment by jlaska** [comment]
> For 'hvac_mode', using the style option has no effect at all, even when setting "active: false".

+1  Using the example provided in the `README.md`, I'm not able to observe any changes to the style of the button.

```yaml
- type: custom:mini-climate
  entity: climate.my_ac
  hvac_mode:
    style: "(value, entity) => ({ color: 'black' })"
```

**Comment by regevbr** [comment]
fix is released in 2.0.3
If you want to override the styles of the hvac when the entity is 'on' you need to use the `!important modifier. You can apply the same fix in buttons, or set the `active` property to always false. In both cases, you need to dynamically decide on the style based on the entity state yourself (entity is given to you as a parameter)
@jlaska and @mooisjken I would appreciate your feedback on the released version.

**Comment by jlaska** [comment]
@regevbr thanks for the response!  Tested `2.0.3` and confirmed the `!important modifier` allows me to use`hvac_mode.style`.  👏
